pom.xml

```diff
@@ -1,21 +1,25 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <parent>
-    <groupId>org.jodreports.poms</groupId>
-    <artifactId>jodreports-parent</artifactId>
-    <version>1</version>
-  </parent>
+<!--   <parent> -->
```

Comment:
I'm uncertain whether a official Maven release would still work, but we should at least try to use the old Maven coordinates.